mibidata/mibi_image.py

```diff
@@ -219,6 +219,18 @@ def __getitem__(self, channels):
         """Equivalent to :meth:`~slice_data`."""
         return self.slice_data(channels)
 
+    def __repr__(self):
+        s = f"{type(self)}\n"
+        for key, val in self.metadata().items():
+            s += f'{key}: {val}\n'
```

Comment:
For a dict of this length it's not a big deal, but in general it's better to do
```
'\n'.join(f'{key}: {val}' for key, val in im.metadata().items())
```
because `+=` creates a new string at each iteration.

Reply:
done